src/visualization/reducers/index.ts

```diff
@@ -24,7 +25,8 @@ const reducers = combineReducers({
     inactive: InactiveReducer,
     error: ErrorReducer,
     fullpage: FullPageReducer,
-    notfound: NotFoundReducer
+    notfound: NotFoundReducer,
+    
 });
```

Comment:
revert this file

Reply:
not resolved - if you disagree mark as won'tfix and give an explanation

---
In reply to: [175853932](https://github.com/Microsoft/clarity-tools/pull/41#discussion_r175853932) [](ancestors = 175853932)